test/e2e/multihoming.go

```diff
@@ -525,6 +588,92 @@ var _ = Describe("Multi Homing", feature.MultiHoming, func() {
 				true, // collocated on the same node
 				Label("STORY", "SDN-5345"),
 			),
+			// The second setup we test configures: a localnet that uses a VLAN, an external router
+			// that acts as gateway for the localnet pod for traffic destined to the host network.
+			// We implement the external router as an external container, where we create a VLAN interface
+			// on top of eth0 and assign to it an IP in the subnet in use by the localnet.
+			//
+			//                            +-----------------------+
+			//                            |     Kubernetes Node   |
+			//                            |       ovn-worker2     |
+			//                            |                       |
+			//    podA (10.244.1.10/24)---+-------[ br-int ]------+--- podL (172.20.0.4/16, net1)
+			//    (default net)           |           |           |     (localnet, VLAN 10)
+			//                            |       [ br-ex ]       |
+			//                            |        172.18.0.2     |
+			//                            +-----------|-----------+
+			//                                        |
+			//                                  host network
+			//                                  172.18.0.0/16
+			//                                        |
+			//                           +------------------------+
+			//                           |     external router    |
+			//                           |                        |
+			//                           |  eth0: 172.18.x.x      |
+			//                           |  eth0.10: 172.20.0.55  |
+			//                           +------------------------+
+			//
+			// Packet path (ping podA → podL):
+			//   podA (10.244.1.10)
+			//     → br-int
+			//       → br-ex (172.18.0.2, SNAT to node IP)
+			//         → eth0 (external router, 172.18.x.x)
+			//           → eth0.10 (external router, 172.20.0.55)
```

Comment:
how does the `ip r` output and `ip a` output of this external router look like?
I assume you added some static routes to back into br-ex for the destination 172.20.0.4?